packages/noports_core/lib/src/sshrv/sshrv_impl.dart

```diff
@@ -73,13 +73,18 @@ class SshrvImplDart implements Sshrv<SocketConnector> {
     try {
       var hosts = await InternetAddress.lookup(host);
 
-      return await SocketConnector.socketToSocket(
+      SocketConnector socketConnector =  await SocketConnector.socketToSocket(
         socketAddressA: InternetAddress.loopbackIPv4,
         socketPortA: localSshdPort,
         socketAddressB: hosts[0],
         socketPortB: streamingPort,
         verbose: true,
       );
+
+      // TBD - Gary/Xavier/Jagan
+      // Should we expose the sockets from socketConnector? whats the best way to do this.
+      //authenticationProvider?.authenticate(socketConnector.socketB);
+      return socketConnector;
```

Comment:
I am mostly unopinionated on this, but we should have some way to kill the socket connector externally, I have thought about some use-cases where this is useful.

Reply:
one can call close() on a socket connector